src/main/java/net/snowflake/ingest/streaming/internal/BlobMetadata.java

```diff
@@ -54,6 +62,39 @@ byte getVersionByte() {
     return bdecVersion.toByte();
   }
 
+  // TODO: @rcheng question - do these need @jsonproperties?
```

Comment:
The above methods aren't called in a Json context, so why do they need @JsonProperty translation? Would the properties that I add need the tag as well?